### PR TITLE
chore: fix relative import

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #4015 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011026&bid=16527929)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011026&bid=16527929)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=5011026&bid=16527929)

File: app/initializers/extensions.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import '../extensions/ember-table/component';
+import 'open-event-frontend/extensions/ember-table/component';
```
Comment (kushthedude) [inline app/initializers/extensions.js]: Are you sure there should not be `initialisers` before the extension in here @maze-runnar ?

**Pull request merged**